3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/outputs/BadRow.scala

```diff
@@ -56,7 +73,7 @@ case class BadRow(
    */
   def toJValue: JValue =
     ("line"           -> line) ~
-    ("errors"         -> errors.toList) ~
+    ("errors"         -> errors.toList.map(_.toString)) ~
```

Comment:
When combining the bad row classes for hadoop shred (which uses ProcessingMessages) and common enrich (which uses Strings) I originally converted all the ProcessingMessages to Strings. But I agree that it would be better to do it the other way round. I could instead change Common Enrich's Strings to ProcessingMessages like this:

```scala
new ProcessingMessage().setMessage(errorString).setLogLevel(LogLevel.ERROR)
```

Does that sound good @alexanderdean ?